docs/nl_NL/settings.md

```diff
@@ -18,3 +18,26 @@ Als u niet-opgeslagen wijzigingen heeft, kunt u de instellingen niet sluiten, u
 ::: danger Reset de instellingen
 Let op: Als u instellingen reset, kunnen ze niet worden hersteld zonder voorafgaande export!
 :::
+
+
+## Moment.js
+This section provides some information on how to configure the date-time configs, e.g. in the module [clock](modules/clock.md).
```

Comment:
English in the dutch version?

Reply:
@JRH-1997 is already informed about missing NL translations ;)